client/x/evmengine/keeper/keeper.go

```diff
@@ -56,6 +58,8 @@ type Keeper struct {
 		Height    uint64
 		UpdatedAt time.Time
 	}
+
+	isExecEngSyncing bool
```

Comment:
Just for learning the code base, this keeper is always on a single go routine ?

Reply:
Right!